### DISCUSSION TITLE
🐛 BUG: Thumbnail seems to be cut out in posts view

### DISCUSSION THREAD

**Discussion by AnishDe12020** [dailydotdev/daily#459] [Feature Requests]
### What went wrong? 🤔

Thumbnail seems to be cut out - 
![image](https://user-images.githubusercontent.com/63192115/150803963-0ace35b6-06b4-41fc-a5fd-90c9607d21a2.png)

### Expected Behavior

The whole thumbnail should be visible

### Actual Behavior

The thumbnail is not completely visible, seems as if there is an issue in the aspect ratio as well.

### Steps to Reproduce Issue

Open any post in the post view

### Solution

Making the thumbnail fully visible

### Environment

Web app

#### Operating System
Linux

#### Version of Daily

Web version so latest

#### Browser

Edge v99, Firefox

**Comment by idoshamun** [comment]
Hi,

This is part of our new design for the article page. Still considering it though.
I'll let you know our final decision :)
